### PR TITLE
RSSI to display 100 instead of SYM_MAX

### PR DESCRIPTION
**Pull request by dzaro-dev**
### CURRENT BEHAVIOUR

The current way the RSSI is displayed is such that the RSSI takes up two characters. Whilst this is OK for most OSD systems which have a dedicated icon for RSSI and LQ, it causes alignement issues with regards to the values when used with the O3 system for example. As the LQ hovers around 100 most of the time and the RSSI is usually lower than 100, there is logic behind the value taking up 2 spaces and being replaced by the MAX symbol when it is 100. 

But this is not ideal when you would like to align those values to the symbol as well as the number

### DESIRED BEHAVIOUS
I think that the MAX symbol should not be used. Istead, the value should be shown as 100 when it is 100, same as the LQ. This way both the LQ and the RSSI, which are most frequently used together will both be showing with the same total number of characters, i.e. 4 in total, 1 for the signal bars icon and 3 for the value. This way users can correclty align the two signal icons one under the other and the values can be aligned to the right. 

### EXAMPLE

In other words, like this:

Screenshot_2024-12-23_at_22 06 36

Specifically, for users of the O3 system, which lacks separate icons for RSSI and LQ can use the custom fields to display what they would like as plain text next to the respective line. 

### OTHER COMMENTS

Other systems would benefit from this as well, as the value will be correclty alligned when placed one under the other.

In my case the LQ is the top and the the RSSI is at the bottom and I added those lables as text next to the value

### ADDITIONAL WORK

The only thing that would need to be done would be to adjust the lenght in the configurator as well.

Feel free to let me know if you think I've made a mistake, but this just my way of thinking about this.

### CODE REVIEW

**Comment by MrD-RC** [comment]
This change would affect _all_ OSDs. Not just DJI. This would cause alignment issues on every system that isn't DJI O3. 

Given that DJI have started actually doing things semi-correctly. And are implementing an INAV font for O4. There is a chance that this will also be implemented in O3. Meaning that the whole DJI compatibility mode can be scrapped and code put back without the hacks.

**Comment by dzaro-dev** [comment]
Alright, as you wish, maybe it's just a personal preference thing. 

I'm waiting for that OSD to come out finally because I exchanged over 20 emails with them and spoke to people from their dev team (presumably, that's what they represented themselves as) in China asking kindly for their support and explaining to them that we need them to either implement the INAV font in the Goggles or just provide a way to load fonts via the MicroSD card directly. 

As you probably know more about this anyway, I'll leave this PR for now, we can revisit it if there is a need inthe future.
